code/modules/mob/mob.dm

```diff
@@ -1538,3 +1538,4 @@ GLOBAL_LIST_INIT(holy_areas, typecacheof(list(
 	. = stat
 	stat = new_stat
 	SEND_SIGNAL(src, COMSIG_MOB_STATCHANGE, new_stat, .)
+var/wait = world.time //This varible allows you to set a cooldown on an ability, for an example usage reference the raven.dm, last line.
```

Comment:
please scope this to 
`/mob/living/simple_animal/pet/raven` and name it something like `caw_cooldown` or whatnot. This is currently scoped **globally** but will only ever been used for the raven at the moment.

Reply:
Localized in the most recent commit!